recipes/r-animation/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = '2.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-animation
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: animation_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/animation_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/animation/animation_{{ version }}.tar.gz
+  sha256: b232fef1b318c79710e5e1923d87baba4c85ffe2c77ddb188130e0911d8cb55f
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('animation')"  # [not win]
+    - "\"%R%\" -e \"library('animation')\""  # [win]
+
+about:
+  home: https://yihui.name/animation
+  license: GPL
```

Comment:
Is this correct?

Reply:
The package is GPL'd: https://github.com/yihui/animation/blob/master/DESCRIPTION#L31

@Cashalow Why did you manually add a Creative Commons license file?

Reply:
I thought this was standard procedure for conda-forge (https://github.com/conda-forge/staged-recipes/pull/4310#issuecomment-343915208), it's just that I must have messed up somewhere for adding a CC instead of GPL file ! Sorry this has been fixed

Reply:
@Cashalow No worries. The license aspect can be confusing and best practices can change. For now for GPL'd R packages we are adding the `license_file` lines. This results in the GPL license file shipped with r-base to be added to the conda package for the recipe.